### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by codesome**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This release uses Write-Ahead Logging (WAL) for the remote_write API. This currently causes a slight increase in memory usage, which will be addressed in future releases.
 
+In 2.7.x, default time retention was used even when size based retention was specified. In this release, default time retention is used only when no size based retention is specified.
```
Comment (brian-brazil) [inline CHANGELOG.md]: This is a CHANGE, list it as such.

Comment (codesome) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 This release uses Write-Ahead Logging (WAL) for the remote_write API. This currently causes a slight increase in memory usage, which will be addressed in future releases.
 
+* [CHANGE] Default time retention is used only when no size based retention is specified. #5216
```
Comment (brian-brazil) [inline CHANGELOG.md]: I'd mention that this is a flag, and mention the flag names. It'll make it easier for users to realise what's going on.

Comment (roidelapluie) [reply CHANGELOG.md]: should we then mention the default value?

Comment (codesome) [reply CHANGELOG.md]: Default values are from the start, I don't think that I required.

**Pull request merged**